src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayRentCost = 40;
```

Comment:
Good job!
[Checklist] It had better change the variables' names on upper case letters

![image](https://github.com/mate-academy/js_task-transportation-on-vacation/assets/72705330/cf018f85-7ffa-4cac-b5ba-39c939cfb4bc)

Reply:
According to Misha's video on the platform, we should use upper-case letters for constants :)

Reply:
agreed with [comment](https://github.com/mate-academy/js_task-transportation-on-vacation/pull/2869/files#r1226322815)
better to use upper-case letters for constants. it's like a rule for the devs